Sources/Sentry/SentryNSURLSessionTaskSearch.m

```diff
@@ -0,0 +1,68 @@
+/**
+ * Part of this code was copied from
+ * https://github.com/AFNetworking/AFNetworking/blob/4eaec5b586ddd897ebeda896e332a62a9fdab818/AFNetworking/AFURLSessionManager.m#L349-L418
+https://github.com/AFNetworking/AFNetworking/blob/4eaec5b586ddd897ebeda896e332a62a9fdab818/LICENSE
+ * under the MIT license
+ */
+
+#import "SentryNSURLSessionTaskSearch.h"
+#import <objc/runtime.h>
+
+@implementation SentryNSURLSessionTaskSearch
+
+/**
+ * WARNING: This code can easily lead to bad behavior, like crashes or messing up HTTP requests. Be
+ * careful when changing it.
+ */
++ (NSArray<Class> *)urlSessionTaskClassesToTrack
```

Comment:
`m`: I know we copied this code from AFNetworking, but can we please add a couple of tests for it? It would be nice if the tests check different iOS versions. You can checkout `testNSURLSession_TraceHeaderAdded` for inspiration.